store/tree.go

```diff
@@ -203,6 +217,10 @@ func (t Tree) SliceAcc(start []byte, end []byte) uint64 {
 	return leftexact + leftrest - rightest
 }
 
+func (t Tree) SplitAcc(key []byte) (uint64, uint64, uint64) {
```

Comment:
@mconcat can we delete this method, and switch testing to use subset accumulation instead?